ffs/integrationtest/integration_test.go

```diff
@@ -768,6 +764,8 @@ Loop:
 }
 
 func TestRenewWithDecreasedRepFactor(t *testing.T) {
+	// Too flaky for CI.
+	t.SkipNow()
```

Comment:
Too flaky to my taste. I prefer not bumping the CI machine price just to make this smooth.